test/config.json

```diff
@@ -2,7 +2,7 @@
   "tests": [
     {
       "file": "test/specs/chrome.js",
-      "timeout": 20000,
+      "timeout": 30000,
```

Comment:
Are the UI tests still only passing with an increased timeout?

Reply:
I'm trying to revert it back to `20000`, let's see if it works.

Reply:
🤞

Reply:
@kopijunkie it takes 3 tries to pass, not bad but it seems that the `face` scenario is the heaviest test suite we have.

Reply:
Probably because of a combination of the tests have to run through most of the steps and test for different config/camera availability scenarios.